lib/tests/dbconnector.test.js

```diff
@@ -0,0 +1,44 @@
+import dbConnector from "../integration/dbConnector";
+
+jest.mock("pg", () => {
+  const pgClient = {
+    connect: jest.fn(),
+    query: jest.fn(),
+    end: jest.fn(),
+  };
+  return { Client: jest.fn(() => pgClient) };
+});
+
+describe("Testing dbConnector", () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("Should create and return a valid client", async () => {
+    const client = dbConnector("postgres://test:test@localhost:5432/test");
+    // Try to connect
+    client.connect();
+    expect(client.connect).toBeCalledTimes(1);
+    // Verify query method
+    client.query("SELECT bearer_token FROM templates;", []);
```

Comment:
Hmmm don't see a point in this considering at this point you're just validating your mock object. I think the connect was enough to do that but this is minor